packages/internals/src/__tests__/handlePanic.test.ts

```diff
@@ -119,14 +134,20 @@ describe('handlePanic', () => {
       'test-message',
       rustStackTrace,
       'test-request',
-      ErrorArea.INTROSPECTION_CLI, // area
+      ErrorArea.LIFT_CLI, // area
```

Comment:
Not sure what we want to do here, it's still about introspection but the code leaves in the migration engine.

Reply:
The `RustPanic` in `MigrateEngine` are statically defined as `LIFT_CLI`. Although it's technically introspection-related, this tells us the engine that was used under the hood. Besides, this `ErrorArea` label is of internal-use only

Reply:
`INTROSPECTION_CLI` would be better to keep for the meaning of the error was actually triggered from the "introspection part of the CLI"
But since it would be complicated and would not provide any benefit, we can go for `LIFT_CLI`
Technically, we could also remove completely the `ErrorArea` one day (but we need to handle that in the error reporting backend)